### ISSUE TITLE
Enhancement: Update Algorithm to create default Avatar for Pàdé

### ISSUE THREAD

**Issue by gjaekel** [igniterealtime/pade#253]
Update the algorithm for choosing the Initials for the default avatar introduced in "OFMeet". This will respect the common European syntax for display names ("lastname, firstname" -> "FL")

**Comment by deleolajide** [comment]
See https://github.com/igniterealtime/pade/blob/f828f42f2708b1ec5c7a0dd26f2b39ea614e8e0d/docs/inverse/index.js#L1749

**Comment by gjaekel** [comment]
I see, source for Pàdé is at this repo. :smile:

**Comment by gjaekel** [comment]
I found the identical code at https://github.com/igniterealtime/pade/blob/f828f42f2708b1ec5c7a0dd26f2b39ea614e8e0d/docs/js/background.js#L1591 ?!? Is this used or is it an artefact?

**Comment by deleolajide** [comment]
> ?!? Is this used or is it an artefact?

Yes. https://github.com/igniterealtime/pade/blob/master/docs-1.6.13.3.zip

**Comment by gjaekel** [comment]
But I don't change `js/background.js::createAvatar()`, just `inverse/index.js::_createAvatar()` to see the expected results.
